pkg/kubelet/dockertools/manager.go

```diff
@@ -61,6 +61,7 @@ const (
 	DockerType = "docker"
 
 	minimumDockerAPIVersion = "1.20"
+	minimumDockerVersion    = "1.8.0"
```

Comment:
I think declaring minimumDockerVersion here is a bit misleading since we don't check it. Let's just make this a constant in the fake docker client for now.